scripts/3_create_v2.sql

```diff
@@ -0,0 +1,57 @@
+create table aws_region
+(
+    id     serial primary key,
+    region varchar not null unique
+);
+
+create table aws_account
+(
+    id      serial primary key,
+    account varchar not null unique
+);
+
+
+create table aws_resource_type
+(
+    id            serial primary key,
+    resource_type varchar not null unique
+);
+
+create table aws_resource
+(
+    id                   bigserial primary key,
+    arn_id               varchar not null, /* nb, this is NOT full arn, it is resource ID like i-5325235 as ARN is redundant and can be reconstructed */
+    aws_account_id       int     not null, /* the account, type and region are immutable, so the lookup tables are referenced as the part of resource, not event */
+    foreign key (aws_account_id) references aws_account (id),
+    aws_region_id        int     not null,
+    foreign key (aws_region_id) references aws_region (id),
+    aws_resource_type_id int     not null,
+    foreign key (aws_resource_type_id) references aws_resource_type (id),
+    meta                 JSONB /* this is not optimal and is not correct as tags can change and changes need tracking */
```

Comment:
question: what lead you to choose JSONB here? what do you think could be more optimal in the future? (or does that need further investigation)

Reply:
Nothing. This part is unchanged from current schema. We will refactor it later, I just wanted to limit the scope of changes, and we do not need to normalize tags for M1.

Reply:
re: optimal in the future - we need to track the changes to the specific tags, possibly in  separate tables as they are mutable and should have their own not_before and not_after values. At least those that establish ownership and tell which CF stack or micros service the resource is part of are of interest. So we will at some point need to stop treating the tags as immutable attribute of AWS resource. Possibly in m2 (depending on if we will be able to do attribution based on "blob" tags fast enough or not).